layouts/default.vue

```diff
@@ -51,6 +51,30 @@
     background-color: #f0f0f0;
   }
 
+  .wrapper {
+    background-color: #f0f0f0;
+    margin: 0 auto;
+    padding: 0;
+    width: 100%;
+    min-height: 100vh;
+    display: flex;
+    justify-content: left;
+    align-items: flex-start;
+    text-align: left;
+  }
+
+  .title {
+    font-family: 'Open Sans', Helvetica, Arial, sans-serif;
+    display: block;
+    font-weight: 400;
```

Comment:
I see that it's impossible to get it to look exactly the same, but I'd go with font-weight 300 here. It looks better and closer to the original in the AngularJS app